src/Generator/Driver.cs

```diff
@@ -452,13 +449,16 @@ public static void Run(ILibrary library)
             if (!driver.Options.DryRun)
             {
                 driver.SaveCode(outputs);
-                if (driver.Options.IsCSharpGenerator && driver.Options.CompileCode)
+                if (driver.Options.IsCSharpGenerator && driver.Options.CompileCode &&
+                    !driver.Options.GenerateDebugOutput)
```

Comment:
What's the reason for this change?

Reply:
Code with debug output is not compilable as debug output is not valid c# syntax.

Reply:
That should not be happening, do you have an example of generated debug output that does not compile?

Reply:
[Urho3D.cs.txt](https://github.com/mono/CppSharp/files/1543388/Urho3D.cs.txt)